From: Miaohe Lin <linmiaohe@huawei.com>
Subject: memory tier: remove unneeded disable_all_demotion_targets() when !CONFIG_MIGRATION
Date: Tue, 6 Jun 2023 20:07:24 +0800

There's no caller of disable_all_demotion_targets() when CONFIG_MIGRATION
is disabled.  Remove it.

Link: https://lkml.kernel.org/r/20230606120724.208552-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memory-tiers.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/memory-tiers.c~memory-tier-remove-unneeded-disable_all_demotion_targets-when-config_migration
+++ a/mm/memory-tiers.c
@@ -451,7 +451,6 @@ static void establish_demotion_targets(v
 }
 
 #else
-static inline void disable_all_demotion_targets(void) {}
 static inline void establish_demotion_targets(void) {}
 #endif /* CONFIG_MIGRATION */
 
_
